src/components/Calculator.js

```diff
@@ -0,0 +1,36 @@
+function Calculator() {
+  return (
+    <div className="calculator">
+      <div className="display">
+        <span>0</span>
+      </div>
+      <div className="keyboard">
+        <div className="digits">
+          <button type="button">AC</button>
+          <button type="button">+/-</button>
+          <button type="button">%</button>
+          <button type="button">7</button>
+          <button type="button">8</button>
+          <button type="button">9</button>
+          <button type="button">4</button>
+          <button type="button">5</button>
+          <button type="button">6</button>
+          <button type="button">3</button>
+          <button type="button">2</button>
+          <button type="button">1</button>
+          <button id="zero" type="button">0</button>
+          <button type="button">.</button>
+        </div>
+        <div className="operators">
+          <button type="button">/</button>
+          <button type="button">x</button>
+          <button type="button">-</button>
+          <button type="button">+</button>
+          <button type="button">=</button>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+export default Calculator;
```

Comment:
- [ ] Kindly let make the Calculator component a class-based component for now, as mentioned in the project requirements below 👇 
![image](https://user-images.githubusercontent.com/55465133/138003988-2e3faaa6-fd3b-42d8-9d58-821e7afa2d91.png)